module/Activity/Module.php

```diff
@@ -52,12 +54,27 @@ public function getServiceConfig()
                 'activity_doctrine_em' => function ($sm) {
                     return $sm->get('doctrine.entitymanager.orm_default');
                 },
+                'activity_form_activity' => function ($sm) {
+                    /** @var \Decision\Service\Member $userService */
+                    $memberService = $sm->get('decision_service_member');
+                    /** @var \User\Model\User $identity */
+                    $identity = $sm->get('user_role');
+
+                    // This is a necessary hack, since the user and corresponding member model is cached
+                    // I could not get the organ from the Member, because this resulted in all kinds of exceptions
+                    // This forces doctrine to fetch a clean object from the database
+                    $user = $memberService->findMemberByLidNr($identity->getLidnr());
+
```

Comment:
Is there any need to save some queries? I do not see any performance issues yet. To quote someone famous: "Premature optimization is the root of all evil." That is also why I disagree with caching the member object, since it probably does not have a lot of influence on server load/rendering time, and it has all sorts of complications (this piece of code is one of them)